streamz/tests/test_graph.py

```diff
@@ -24,6 +24,21 @@ def test_create_graph():
         assert t in g
 
 
+def test_create_cyclic_graph():
```

Comment:
Would this have caused infinite recursion before?

Reply:
Yes, this would have failed.